### ISSUE TITLE
region geojson files do not parse properly in standard tools

### ISSUE THREAD

**Issue by matthewhoffman** [MPAS-Dev/geometric_features#2]
I found that geojson files in this repo would not display properly in any off-the-shelf tools I tried:
1. QGIS
2. http://geojson.io/
3. http://geojsonlint.com/
4. http://jansanchez.com/geojson-viewer/
5. https://developers.google.com/maps/documentation/javascript/examples/layer-data-dragndrop

I spent a little while trying to figure out if there was mismatched brackets or something and could not find anything wrong.  I also tried stripping out newlines, tabs, and even spaces but that did not help.  I tried a few of the landice regions, and I tried a few of the ocean regions, even reducing them to a dozen or so points (in case any of those tools couldn't handle large data).

Anyway, as long as our MPAS tools work with format, this isn't a big deal, but it would be nice to quickly visualize or validate files with other off-the-shelf tools.

**Comment by xylar** [comment]
@matthewhoffman, I had trouble using shapely within python as well.  I found that the problem was that the coordinates as they were being written out had one less level of square brackets compared with the geojson standard.  I think Doug as just fixed this.  See if you can view the individual region files in standard tools now.  If not, let me know and I'll keep looking into what isn't quite right here.

**Comment by douglasjacobsen** [comment]
@xylar and @matthewhoffman You can actually see that this has been fixed by navigating here: https://github.com/MPAS-Dev/geometric_features/blob/master/ocean/region/Adriatic_Sea/region.geojson

**Comment by xylar** [comment]
GitHub is awesome! @douglasjacobsen, thanks for fixing this.

By the way, here is one of the regions that needs to be fixed because of the -180/180 split.  
https://github.com/MPAS-Dev/geometric_features/blob/master/landice/region/Antarctica_IMBIE17/region.geojson

Is it also going to be a problem when latitudes cross +/- 90?  Let's say we started out with a circle around -75 latitude.  Now, let's say we have split that circle into two pieces already. Presumably we can't just have a between `[180,-75]` and `[0, -75]`.  Instead, I'm guessing we need a line like:

`[[180,-75],[180,-90],[0,-90],[0,-75]]`

In other words, polygons that contain a pole will need to be a special case within the special case of polygons that cross +/-180.  Does that sound right?

**Comment by douglasjacobsen** [comment]
@xylar, I'm not 100% sure what has to happen when a region includes a pole. I think your best bet is to try the region in the mask creator tool to see what happens.

However, here is a features file that contains all of the ocean regions. You can click on a specific one to see what it's name is, and maybe that would help with determining what actually has to happen.
https://gist.github.com/douglasjacobsen/90a77c7023d0eb1cec80

**Comment by xylar** [comment]
@douglasjacobsen, that's perfect.  I was able to see that the Arctic basin:
https://raw.githubusercontent.com/MPAS-Dev/geometric_features/master/ocean/region/Arctic_Ocean/region.geojson
has the following four points, consistent with my guess about how the pole needs to be treated:

```
[ -180.000000, 71.535845],
[ -180.000000, 90.000000],
[ 180.000000, 90.000000],
[ 180.000000, 71.535855],
```

I will make sure the tool I write handles this.

@matthewhoffman, if you agree that Doug's changes make the files compatible with other tools, you maybe want to close this issue. and I'll stop using it for unrelated discussions.

**Comment by matthewhoffman** [comment]
That does indeed fix the issue for all of the tools I had experimented with!  Thanks @douglasjacobsen (and @xylar for pointing it out).  At least now we have a nice list of web tools for plotting this - though github's rendering is just as good and more convenient!
